### PR TITLE
added HTML

### PR DESCRIPTION
**Pull request by BeeNorrg**

### CODE REVIEW

File: frontend_test/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (noahschlag) [inline frontend_test/index.html:1]: yessir

Comment (noahschlag) [inline frontend_test/index.html:1]: s

File: frontend_test/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en">
```
Comment (noahschlag) [inline frontend_test/index.html:2]: a

File: frontend_test/index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+<head>
```
Comment (noahschlag) [inline frontend_test/index.html:3]: s

File: frontend_test/index.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <meta charset="UTF-8">
```
Comment (noahschlag) [inline frontend_test/index.html:4]: s

File: frontend_test/index.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (noahschlag) [inline frontend_test/index.html:5]: s

File: frontend_test/index.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (noahschlag) [inline frontend_test/index.html:6]: s

File: frontend_test/index.html [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    <!-- Bootstrap -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <!-- Local CSS -->
+    <link rel="stylesheet" type="text/css" href="assets/style.css"> 
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (noahschlag) [inline frontend_test/index.html:16]: s

Comment (CalvinJacobsen) [inline frontend_test/index.html:16]: looks good

**Review by CalvinJacobsen — APPROVED** [review]

**Review by noahschlag — APPROVED** [review]

**Review by noahschlag — APPROVED** [review]

**Review by jimnguyen220 — APPROVED** [review]

**Pull request merged**